mailpoet/lib/Newsletter/NewsletterResendController.php

```
@@ -0,0 +1,332 @@
+    );
+
+    /** @var list<string|int> $column */
+    $column = $result->fetchFirstColumn();
+    return array_map('intval', $column);
+  }
+
+  /**
+   * @param int[] $nonOpenerIds
+   * @param int[] $segmentIds
+   */
+  private function addNonOpenersToTask(ScheduledTaskEntity $task, array $nonOpenerIds, array $segmentIds, ?int $filterSegmentId): int {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$filterSegmentId'. (undefined)

(UnusedFormalParameter)